spec/support/common_actions/documents.rb

```diff
@@ -0,0 +1,52 @@
+module Documents
+  def do_login_for(user)
+    common_do_login_for(user, management: management)
+  end
```

Comment:
Nice catch! 

> By the way, I've no idea why this method but it's magically using the management variable even if we're not passing it as an argument, although it seems to work properly ❓.

After some debugging, it seems the `management` variable gets its value from the caller context.

> It's very possible that we already had this very same problem, since previously we were defining the methods inside the default object...

Yes, I think so. I finally left one method within the `common_actions/users` module and added to all the method calls the management argument, so we do not have to wonder where it comes from.